src/grpc/lib-oc/OpenConfigNpuUtilization.cpp

```diff
@@ -41,25 +41,19 @@ OpenConfigNpuUtilization::iterate (JuniperNetworksSensors *handle, Telemetry::Op
 
             kv = datap->add_kv();
             kv->set_key("memory/" + mem.name() + "/lowest_util");
-            kv->set_int_value(mem.highest_util());
+            kv->set_int_value(mem.lowest_util());
 
-            if (mem.average_cache_hit_rate()) {
-                kv = datap->add_kv();
-                kv->set_key("memory/" + mem.name() + "/average_cache_hit_rate");
-                kv->set_int_value(mem.average_cache_hit_rate());
-            }
+            kv = datap->add_kv();
+            kv->set_key("memory/" + mem.name() + "/average_cache_hit_rate");
+            kv->set_int_value(mem.average_cache_hit_rate());
 
-            if (mem.highest_cache_hit_rate()) {
-                kv = datap->add_kv();
-                kv->set_key("memory/" + mem.name() + "/highest_cache_hit_rate");
-                kv->set_int_value(mem.highest_cache_hit_rate());
-            }
+            kv = datap->add_kv();
```

Comment:
Ditto. Any particular reason you believe we don't have to check for non zero value condition ?